ironfish/src/rpc/routes/node/getStatus.ts

```diff
@@ -143,6 +150,15 @@ export const GetStatusResponseSchema: yup.ObjectSchema<GetNodeStatusResponse> =
       .object({
         size: yup.number().defined(),
         sizeBytes: yup.number().defined(),
+        maxSizeBytes: yup.number().defined(),
+        saturation: yup.number().defined(),
```

Comment:
Could probably remove this saturation since we're not using it

Reply:
Is this done? It's not clear to me why this is in the yup schema but not in the TS GetNodeStatusResponse type.

Reply:
yes, removed